woocommerce/woocommerce-ios
# Variations: Edit Attributes

Closes #3515, #3516.

## Description

This PR adds second screen of variation attributes edit flow (options picker), connects it with attributes list and with data update.

## Changes

- Added `AttributeOptionListSelectorCommand` for `ListSelectorViewController` to display options for picked attribute.
- Added "Any Attribute" options on client side.
- Added view model for `AttributePickerViewController`.
- Added unit tests for `AttributePickerViewModel` and `AttributeOptionListSelectorCommand`.

## Testing

1. Open product with existing variations
2. Open variations details
3. Tap on attributes cell (added in #3519)
4. Tap any attribute and change it
5. Go to previous screen
6. Observe "Done" button appears only if any attribute is changed
7. Tap "Done" and then "Update" to save new attributes

Try full flow with changing to "Any Attribute" and from "Any Attribute".

## Screenshots

Attributes | Options
--|--
![Simulator Screen Shot](https://user-images.githubusercontent.com/3132438/107056423-61bdd500-67e3-11eb-9489-1bb0f44229ad.png)|![Simulator Screen Shot](https://user-images.githubusercontent.com/3132438/107056433-64202f00-67e3-11eb-8cd1-55b7e7b7bade.png)

## Video

https://user-images.githubusercontent.com/3132438/107057038-0dffbb80-67e4-11eb-89b0-ab92d7b2da40.mp4


Update release notes:
